javascript/javascript3/week3/index.js

```diff
@@ -0,0 +1,165 @@
+const refs = {
+    input: document.querySelector('[name="website"]'),
+    makeScreenshotBtn: document.querySelector('[name="make-screenshot"]'),
+    saveScreenshotBtn: document.querySelector('[name="save-screenshot"]'),
+    libraryScreenshotsBtn: document.querySelector('[name="library-screenshots"]'),
+    libraryList: document.querySelector('.screenshots-list')
+};
+
+const BASE_URL_SCREENSHOT_API = 'https://website-screenshot6.p.rapidapi.com/screenshot/';
+const KEY_API_SCREENSHOT = 'default-application_7160449';
+
+const BASE_URL_CRUD_CRUD = ' https://crudcrud.com/api';
+const KEY_API_CRUD_CRUD = '5b127da1ec9b44eab72f4d5992846501';
+
+let screenshotUrl = '';
+    
+refs.makeScreenshotBtn.addEventListener('click', onMakeScreenshotBtnClick);
+refs.saveScreenshotBtn.addEventListener('click', onSaveScreenshotBtnClick);
+refs.libraryScreenshotsBtn.addEventListener('click', onLibraryScreenshotsBtnClick);
+refs.libraryList.addEventListener('click', onDeleteBtnClick);
+
+
+
+function onMakeScreenshotBtnClick() {
+    
+    refs.libraryList.innerHTML = "";
+    refs.libraryList.innerHTML = "Wait, it's might take some time";
+
+    fetchScreenshotApi(refs.input.value).then(data => {
+        createOneScreenshotMarkup(data.screenshotUrl);
+        screenshotUrl = data.screenshotUrl;
+        console.log(screenshotUrl);
+        if (screenshotUrl === undefined) {
+            refs.libraryList.innerHTML = `Vi did not find any matches`;
+        }
+    }).catch(error => console.log(error))
+    
+}
+
+
+function onSaveScreenshotBtnClick() {
+    refs.input.value = "";
+    refs.libraryList.innerHTML = "Screenshot saved in library";
+    console.log(screenshotUrl);
+    fetchSavingScreenshot(screenshotUrl).then(data=> console.log(data)).catch(error=>console.log(error))
+
+}
+
+function onLibraryScreenshotsBtnClick() {
+
+    fetchLibraryScreenshots().then(data => {
+        refs.libraryList.innerHTML = "";
+        createLibraryMarkup(data)
+    }).catch(error => console.log(error))
+}
+
+
+function onDeleteBtnClick(event) {
+    if (event.target.classList.contains('delete-btn')) {
+        const id = event.target.dataset.id
+        console.log(id);
+        fetchDeleteScreenshot(id);
+        setTimeout(()=>{ fetchLibraryScreenshots().then(data => createLibraryMarkup(data))
+        .catch(error=>console.log(error))
+    }, 2000)
+       
+        
+  }
+    return
```

Comment:
A return statement on the _last_ line of a function is not needed, since the function will return here anyways.